main.js

```diff
@@ -258,12 +259,15 @@ function join() {
 						server.motd = `Place in queue: ${webserver.queuePlace} ETA: ${webserver.ETA}`; // set the MOTD because why not
 						webserver.ETA = Math.floor(ETAmin / 60) + "h " + Math.floor(ETAmin % 60) + "m";
 						webserver.finTime = new Date((new Date()).getTime() + ETAmin * 60000);
-						if (config.get("userStatus")) {{ //set the Discord Activity
-							if (displayEmail){
-								logActivity("P: " + positioninqueue + " E: " + webserver.ETA + " - " + options.username);
-							} else {
-								logActivity("P: " + positioninqueue + " E: " + webserver.ETA + " - " + client.username);}
-						}} else {
+						if (config.get("userStatus")) {
+							{ //set the Discord Activity
+								if (displayEmail) {
+									logActivity("P: " + positioninqueue + " E: " + webserver.ETA + " - " + options.username);
+								} else {
+									logActivity("P: " + positioninqueue + " E: " + webserver.ETA + " - " + client.username);
+								}
+							}
```

Comment:
```suggestion
						if (config.get("userStatus")) {
							//set the Discord Activity
							if (displayEmail) {
								logActivity("P: " + positioninqueue + " E: " + webserver.ETA + " - " + options.username);
							} else {
								logActivity("P: " + positioninqueue + " E: " + webserver.ETA + " - " + client.username);
							}
```

Reply:
Whats the difference?

Reply:
Oh wait i see it now
I wonder why i have been seeing a lot of duplicate stuff lately.